public/locales/en/transactions.json

```diff
@@ -15,8 +15,11 @@
   "none": "None",
   "bank": "Bank",
   "crisis": "Crisis",
+  "data": "Data",
   "distribution": "Distribution",
+  "ecocredit": "Ecocredit",
   "governance": "Governance",
+  "group": "Group",
```

Comment:
Why add these if you didn't even put them in here https://docs.bigdipper.live/cosmos-based/frontend/customisations/tx#update-message-filter

Reply:
I did put them in src/components/transaction_messages_filter/utils.tsx. Or you mean lack in other file?